OloEngine/src/OloEngine/Renderer/Commands/FrameDataBuffer.cpp

```
@@ -0,0 +1,150 @@
+
+namespace OloEngine
+{
+    FrameDataBuffer::FrameDataBuffer(sizet boneCapacity, sizet transformCapacity)
+    {
+        m_BoneMatrices.resize(boneCapacity);
+        m_Transforms.resize(transformCapacity);
+        m_BoneMatrixOffset = 0;
+        m_TransformOffset = 0;
+    }
+
+    void FrameDataBuffer::Reset()
```

Cppcheck (2.19.0)

[style] The function 'Reset' is never used.

(unusedFunction)